Signal/src/call/OutboundCallInitiator.swift

```diff
@@ -113,4 +150,17 @@ import Foundation
         return true
     }
 
+    private func presentCallInterstitial(_ callToken: String) {
```

Comment:
nit: I would leave this parameter named, e.g. `presentCallInterstitial(callToken: String)`

Idiomatically, as it reads now, I'd expect you're handing it some kind of `CallInterstitial`.